File: beecrowd/1/1012/main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var A, B, C float64
	fmt.Scanf("%f %f %f", &A, &B, &C)
	fmt.Printf("TRIANGULO: %.3f\n", (A*C)/2)
	fmt.Printf("CIRCULO: %.3f\n", 3.14159*math.Pow(C, 2))
	fmt.Printf("TRAPEZIO: %.3f\n", ((A+B)*C)/2)
	fmt.Printf("QUADRADO: %.3f\n", B*B)
	fmt.Printf("RETANGULO: %.3f\n", A*B)
    fmt.Scanf("%f", &A)
}

File: beecrowd/1/1012/main.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

int main() {
    double A, B, C;
    std::cin >> A >> B >> C;
    std::cout << std::fixed << std::setprecision(3)
        << "TRIANGULO: " << (A * C) / 2 << std::endl
        << "CIRCULO: " << 3.14159 * std::pow(C, 2) << std::endl
        << "TRAPEZIO: " << ((A + B) * C) / 2 << std::endl
        << "QUADRADO: " << B * B << std::endl
        << "RETANGULO: " << A * B << std::endl;
}
